addons/surgery/stringtable.xml

```diff
@@ -474,7 +474,7 @@
             <Czech>Vyčistit rány</Czech>
             <French>Débrider les plaies</French>
             <Korean>상처 제거</Korean>
-            <German>Wunddebridement</German>
+            <German>Wunde debridieren</German>
```

Comment:
I would call it "Wundreinigung / Wundausschneidung"

Reply:
I Just went for the spelling.
But if we go there: I would change it to "Wunde Chirurgisch (Chir.) Debridieren", since in this Specific case the Debridement is executed surgically.
Then the action "Schneide/Reinige" would suite as well. (I then would have to change all Debridement actions accordingly).